### PR TITLE
chore(ssa refactor): Add simple constant return program

### PR DESCRIPTION
**Pull request by kevaundray**
# Description

Related to #1418 

## Problem\*

Resolves 

## Summary\*

This PR sets out to

### Example

Before:

```

```

After:

```

```

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

**Review by jfecher — DISMISSED** [review]

**Comment by jfecher** [comment]
Approved, but shouldn't the test fail for this?

**Comment by kevaundray** [comment]
> Approved, but shouldn't the test fail for this?

This branches off of m0 so it should pass now -- m0 was failing because of a clippy error

File: crates/nargo_cli/tests/test_data_ssa_refactor/constant_return/src/main.nr [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// This is a test for simple return with a constant.
+//
+// ACIR generation checks if there is no return 
+// type by checking to see if the return type is a numeric
+// constant with type None.
```
Comment (joss-aztec) [inline crates/nargo_cli/tests/test_data_ssa_refactor/constant_return/src/main.nr:5]: I'm not following this comment - it sounds irrelevant to the test in question. Furthermore I'm yet to find the described check happening anywhere.

Comment (kevaundray) [reply crates/nargo_cli/tests/test_data_ssa_refactor/constant_return/src/main.nr:5]: This is what it was doing before the merge, and I'm guessing the dfg is doing this internally when we call `is_return_type`

Comment (kevaundray) [reply crates/nargo_cli/tests/test_data_ssa_refactor/constant_return/src/main.nr:5]: I'll remove it

File: crates/nargo_cli/tests/test_data_ssa_refactor/constant_return/src/main.nr [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// This is a test for simple return with a constant.
+//
+// 
+// This is just a low hanging test to check this.
+//
```
Comment (kevaundray) [inline crates/nargo_cli/tests/test_data_ssa_refactor/constant_return/src/main.nr:5]:
```suggestion
```

**Pull request merged**